deployment/helm/ditto/Chart.yaml

```diff
@@ -16,7 +16,7 @@ description: |
   A digital twin is a virtual, cloud based, representation of his real world counterpart
   (real world “Things”, e.g. devices like sensors, smart heating, connected cars, smart grids, EV charging stations etc).
 type: application
-version: 3.3.7  # chart version is effectively set by release-job
+version: 3.3.8  # chart version is effectively set by release-job
 appVersion: 3.3.7
```

Comment:
@WatcherWhale could you please update to `appVersion: nightly` - that should fix the akka/pekko issues against master branch